render.yaml

```diff
@@ -4,13 +4,13 @@ services:
 - type: web
   name: blockipedia
   env: static
-  buildCommand: cd frontend; yarn; yarn build
+  buildCommand: make build-frontend
```

Comment:
I usually prefer to not use `Make` everywhere. Make addresses the lack of build tool in C/C++. In frontend, we usually have npm/yarn to do this, and it helps me to recognise immediately what is going on (what tool is doing what) when I see yarn/npm commands rather than make command.

Reply:
For me, I simply love to be able to manage all things from the top-level. So if I'm in the top-level dir, I can just `make build-frontend` without having to `cd` into `frontend` dir. And the rest is just following the "simple source of truth" principle 😀

We could also leave it as `cd frontend; yarn; yarn build` here in this line, but then, in the future if we ever update the `make build-frontend` script, we'll have to remember to update it here in `render.yaml` as well...

Or perhaps a "best of both world" approach: factoring this out into its own shell script, and then in both the `Makefile` and `render.yaml`, we only need to execute that shell script?